whitepaper/whitepaper.md

```diff
@@ -124,12 +133,248 @@ Blockchain technologies are growing in usage, but fragmentation is a big problem
+
+<img src="./ledger_entry_point_coordination.png" width="400"><br>Figure description: Ledger entry point coordination</img>
+
+
+
+
+Legend:
+
+<img src="https://render.githubusercontent.com/render/math?math=O_1 =\text{ Object 1}">
+
+<img src="https://render.githubusercontent.com/render/math?math=O_1^{begin}= \text{ State of object } \quad\!\!\!\! O_1 \quad \!\! \! \! \text{ at the beginning of transaction}">
+
+<img src="https://render.githubusercontent.com/render/math?math=O_2^{begin} = \text{ State of object } \quad\!\!\!\! O_2 \quad\!\!\!\! \text{ at the beginning of transaction}">
+
+<img src="https://render.githubusercontent.com/render/math?math=O_1^{end} = \text{ State of object } \quad\!\!\!\! O_1 \quad\!\!\!\! \text{ at the end of transaction}">
+
+<img src="https://render.githubusercontent.com/render/math?math=O_2^{end} = \text{ State of object } \quad\!\!\!\! O_2 \quad\!\!\!\! \text{ at the end of transaction}">
+
+<img src="https://render.githubusercontent.com/render/math?math=\overline{O}_1^{end} = \text{ Representation of object } \quad\!\!\!\! O_1\quad\!\!\!\! \text{ at the end of transaction in another blockchain}">
+
+<img src="https://render.githubusercontent.com/render/math?math=\overline{O}_2^{end} = \text{ Representation of object } \quad\!\!\!\! O_2 \quad\!\!\!\! \text{ at the end of transaction in another blockchain}">
+
+<img src="https://render.githubusercontent.com/render/math?math=T_1 =\text{ Transaction 1}">
+
+<img src="https://render.githubusercontent.com/render/math?math=T_2 =\text{ Transaction 2}">
+
+<img src="https://render.githubusercontent.com/render/math?math=\overline{T}_1 = \text{ Representation of transaction 1 } \quad\!\!\!\! (T_1) \quad\!\!\!\! \text{ in another blockchain}">
+
+<img src="https://render.githubusercontent.com/render/math?math=\overline{T}_2 = \text{ Representation of transaction 2 } \quad\!\!\!\! (T_2) \quad\!\!\!\! \text{ in another blockchain}">
+
+<img src="https://render.githubusercontent.com/render/math?math=E_1 =\text{ Event 1}">
+
+<img src="https://render.githubusercontent.com/render/math?math=E_2(O_1||T_1||E_1) = \text{ Event 2 depends on }\quad\!\!\!\! O_1 \quad\!\!\!\!\text{ or }\quad\!\!\!\!T_1\quad\!\!\!\! \text{ or }\quad\!\!\!\!E_1">
+
+<img src="https://render.githubusercontent.com/render/math?math=T_2(O_1||T_1||E_1) = \text{ Transaction 2 depends on }\quad\!\!\!\!O_1\quad\!\!\!\! \text{ or } \quad\!\!\!\!T_1\quad\!\!\!\! \text{ or }\quad\!\!\!\!E_1">
+
+	
+
+
+
+### 1.1.3 Burning or Locking of Assets
+
+
+To guarantee unicity, an asset (NFA or FA) has to be burned or locked before being released in another blockchain. Locked assets can be unlocked in case the asset is retransferred back to its original blockchain, whereas the burning of assets is an irreversible process. Burning of assets usually applies more to tokens/coins (FAs)<sup>[h](#12-footnotes)</sup> and can be seen as a donation to the community since the overall value of the currency increases. 
```

Comment:
I included the explanation from the footnote into the main text of the revised write-up and highlighted that burning/locking of assets can be avoided when wallets/accounts are present on both blockchains and even should be avoided in ordinary token exchanges, such as with BTC and ETH, because their minting process can not be leveraged on during a ledger transfer.